samples/boards/arc_secure_services fails on nsim_sem

```
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am the main thread in secure world: 0
Go to normal application
***** Booting Zephyr OS build zephyr-v2.0.0-1024-g30fc3e9f8e2a *****
I am thzephyr/CMakeFiles/run.dir/build.make:57: recipe for target 'zephyr/CMakeFiles/run' failed
CMakeFiles/Makefile2:478: recipe for target 'zephyr/CMakeFiles/run.dir/all' failed
CMakeFiles/Makefile2:485: recipe for target 'zephyr/CMakeFiles/run.dir/rule' failed
Makefile:235: recipe for target 'run' failed
```

--- a/samples/boards/arc_secure_services/sample.yaml
+++ b/samples/boards/arc_secure_services/sample.yaml
@@ -4,6 +4,8 @@ sample:
   name: Designware ARC Secure monitor
 tests:
   sample.board.arc_secure_services:
+    # Requires multiple kernels in an AMP config. See README.rst
+    build_only: true
     platform_whitelist: nsim_sem em_starterkit_em7d_secure
     tags: secure
     harness: console

